.github/workflows/contracts-testing.yml

```
@@ -40,14 +40,14 @@ jobs:
           54.185.253.63:443
 
     - name: Setup Node.js environment
-      uses: actions/setup-node@2a017f350dbf6c4b6bb4508cc83809719115162e
+      uses: actions/setup-node@v4
       with:
-        node-version: 16.x   
+        node-version: 18.x   
```

yamllint

[error] trailing spaces

(trailing-spaces)